documents/git-workflow.md

```diff
@@ -0,0 +1,84 @@
+This document is the best practice guide that contains the rules to follow when working with AsyncAPI repositories.
+
+### Basic rules
+
+Each contributor and maintainer in AsyncAPI must follow this workflow:
+
+* Work on forked repositories.
+* Create branches on the fork and avoid working directly on the `master` branch.
+
+## Prepare the fork
+
+A fork is a copy of the repository from which you raise pull requests to propose changes to the original repository.
+The unified AsyncAPI contribution workflow that bases on forks allows both the members of the Kyma organization and the external contributors to contribute code and content through the same process. This keeps the main repositories branches clean as contributors create branches only on the forked repositories.
+
+## Configure your fork
+
+The document refers to the original repository as the upstream repository and to the forked repository as the origin repository. We assume you already have a fork of the upstream and you `git clone` it laready.
```

Comment:
```suggestion
The document refers to the original repository as the upstream repository and to the forked repository as the origin repository. We assume you already have a fork of the upstream and you `git clone` it already.
```
I mean, `laready` could be a new thing :thinking: :tada:

Reply:
😘